src/ide/completion/path.rs

```diff
@@ -181,3 +164,37 @@ pub fn path_prefix_completions(
         _ => vec![],
     })
 }
+
+fn trait_items_completions(db: &AnalysisDatabase, trait_id: TraitId) -> Vec<CompletionItem> {
```

Comment:
Does it make sense to propose all of those? 
i.e.: why would i want to have impl names or types proposed here?